manifests/config.pp

```diff
@@ -35,6 +35,15 @@
     mode  => '0640',
   }
 
+  $uuid_fact_signature = foreman::hmac_signature($foreman::oauth_consumer_secret, $foreman::instance_id)
+  file { 'foreman_uuid.json':
+    path    => '/opt/puppetlabs/facter/facts.d/foreman_uuid.json',
```

Comment:
Is this a safe path to assume @ekohl ? Or does this need a switcher on AIO similar to puppet_ssldir? (https://github.com/theforeman/puppet-foreman/pull/914/files#diff-e78db79abb0b543310767fb9d2814cc4e2408f613de4985186251c70bdd4edbbR123)

Reply:
According to this there are three paths where facts can be placed by default and those are 
```
/opt/puppetlabs/facter/facts.d/
/etc/puppetlabs/facter/facts.d/
/etc/facter/facts.d/
```

Out of those, only the one in `/opt` is fully present on a foreman machine, for the other two we'd have to manage last two levels of directories.

[1] - https://puppet.com/docs/puppet/7.5/external_facts.html#executable-fact-locations